SingularityUI/app/components/dashboard/MyGroupRequests.jsx

```diff
@@ -0,0 +1,76 @@
+import React, { PropTypes } from 'react';
+import { connect } from 'react-redux';
+
+import { Row, Col, ButtonGroup, DropdownButton, MenuItem } from 'react-bootstrap';
+
+import UITable from '../common/table/UITable';
+import { RequestId, Type, LastDeploy, Actions } from '../requests/Columns';
+import Utils from '../../utils';
+import { SetDashboardGroup } from '../../actions/ui/dashboard';
+import * as RequestsSelectors from '../../selectors/requests';
+
+const MyGroupRequests = ({groupRequests, groups, currentGroup, setCurrentGroup}) => {
+  if (groups.length == 0) {
+    return null;
+  }
+
+  let requestsSection = (
+    <div className="empty-table-message"><p>No requests</p></div>
+  );
+
+  if (groupRequests.length > 0) {
```

Comment:
You can set an `emptyTableMessage` prop on the UITable rather than use this if statement.